### DISCUSSION TITLE
Skybox Image

### DISCUSSION THREAD

**Discussion by begumbeyza** [google/model-viewer#3260] [Q&A]
Hi,I added skybox-image(hdr) but model and environment rotate simultaneously.I only want one of them to move and 
I don't want it to show the bottom of the model.

Thanks for answer.

**Comment by samaneh-kazemi** [comment]
HI @begumbeyza, can you please share a link to your code/use case so we have more context?

**Reply by samaneh-kazemi** [reply]
Also setting max camera orbit for phi to 90 degree is exactly horizontal and you can work with that.
`max-camera-orbit="auto 90deg auto"`
Checkout our documentations for more details on [max-camera-orbit](https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-maxCameraOrbit). Hope this helps.

**Reply by samaneh-kazemi** [reply]
And I recommend looking at our examples for skybox. e.g. [Turn Skybox](https://modelviewer.dev/examples/stagingandcameras/#:~:text=Add%20other%20interactions%20like%20rotating%20the%20skybox)

**Reply by begumbeyza** [reply]
` <model-viewer bounds="tight" src="assets/model.glb"
                skybox-image="assets/back.hdr"
                ar ar-modes="webxr scene-viewer quick-look" 
                auto-rotate="" ar="" exposure="1" camera-controls touch-action="none"   shadow-intensity="4.0"  data-js-focus-visible="" ar-status="not-presenting">`
I use this code.

**Comment by begumbeyza** [comment]
Hi,I added max-camera-orbit="auto 90deg auto" and now don't show the bottom of the model.Thank you.But another problem is still going on.I read and try document.not a smooth image because model and skybox-image are moving.
only skybox-image will move but I want to see the back and front of the model.Is that possible?
Thank you.

**Reply by elalish** [reply]
That's not how it works by default, but `turntableRotation` controls the relative angle between the model and the skybox/lighting, so you can use that to adjust things. Look at the example linked above for inspiration.
